modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceItem.js

```diff
@@ -95,70 +97,99 @@ const ExperienceItem = ({
 		>
 			<ClayList.ItemField expand>
 				<ClayButton displayType="unstyled" onClick={handleSelect}>
-					<strong className="text-truncate">
-						{experience.name}
-
-						{experience.hasLockedSegmentsExperiment && (
-							<>
-								<ClayIcon
-									className="ml-2 text-secondary"
-									onMouseEnter={() => setShowtoolTip(true)}
-									onMouseLeave={() => setShowtoolTip(false)}
-									ref={iconRef}
-									symbol="lock"
-								/>
-
-								{showtoolTip && (
-									<Popover
-										anchor={iconRef.current}
-										header={Liferay.Language.get(
-											'experience-locked'
-										)}
-									>
-										{Liferay.Language.get(
-											'edit-is-not-allowed-for-this-experience'
-										)}
-									</Popover>
+					<ClayLayout.ContentRow verticalAlign="center">
+						<ClayLayout.ContentCol
+							style={{flexShrink: 1, minWidth: 0}}
+						>
```

Comment:
I am not sure if we can/should be using inline styles. Can you check that with infra (aka Marcos)? Personally I like them for this kind of small tweaks, but I've never used them in Liferay

Reply:
I found 47 files with inline styles... 🤔 All of them with maximum of two properties.

Reply:
I think we can go with this 🤷‍♀️